packages/sidenav/test/sidenav.test.ts

```diff
@@ -222,66 +196,41 @@ describe('Sidenav', () => {
         await elementUpdated(el);
 
         expect(el.value).to.equal('Section 0');
-        expect(toBeSelected.tabIndex, '-1 when click based focus').to.equal(-1);
+        expect(toBeSelected.tabIndex, 'will be new focusable child').to.equal(
```

Comment:
I'm honestly not sure how these tests passed previously...like at all.

Reply:
might be worth digging into a touch more on `main` so see if you can get them to fail.